chalicelib_fourfront/checks/wrangler_checks.py

```diff
@@ -1303,14 +1306,28 @@ def users_with_doppelganger(connection, **kwargs):
             a_case['log'] = a_case['log'] + add_on
             a_case['brief'] = a_case['brief'] + add_on
 
-    check.full_output = {'result': cases, 'ignore': ignored_cases}
+    # are the ignored ones getting out of control N.B. Don't think this needs to fail
+    if len(ignored_cases) > 100:
+        fail_msg = 'Number of ignored cases is very high, time to resolve'
+        check.brief_output = fail_msg
+        check.full_output = {'result': [fail_msg, ], 'ignore': ignored_cases}
+        check.status = 'WARN'
+        return check
+
+    check.full_output = {'to_check': cases, 'ignore': ignored_cases, 'close_matches': iffy_cases}
     if cases:
         check.summary = 'Some user accounts need attention.'
         check.brief_output = [i['brief'] for i in cases]
         check.status = 'WARN'
     else:
         check.summary = 'No user account conflicts'
         check.brief_output = []
+
+    # are the ignored ones getting out of control N.B. Don't think this needs to fail
```

Comment:
Duplicated section to 1309-1314?

Reply:
Yes but the bit above was what was left by mistake.  Now removed.